.github/workflows/code_checks.yml

```diff
@@ -0,0 +1,37 @@
+name: Code Checks
+on: [push]
+jobs:
+  linting:
+    runs-on: ubuntu-20.04
+    env:
+      BUNDLE_ENTERPRISE__CONTRIBSYS__COM: ${{ secrets.BUNDLE_ENTERPRISE__CONTRIBSYS__COM }}
+      RAILS_ENV: test 
+
+    steps:
+
+      - uses: actions/checkout@v2
+      
+      - uses: ruby/setup-ruby@v1
+        with:
+          bundler-cache: true
+
+      - name: Install dependencies (pdftk, poppler, imagemagick)
+        run: sudo apt install imagemagick pdftk poppler-utils
+          
+      - name: Install Dependencies
+        run: bundle install --path vendor/bundle
+
+      - name: Run bundle-audit (checks gems for CVE issues)
+        run:  bundle exec bundle-audit check --update
+ 
+      - name: Run Rubocop
+        run: bundle exec rubocop --parallel --format progress --format json --out rubocop.json
+        
+      - uses: reviewdog/action-brakeman@v1
+        with:
+          github_token: ${{ secrets.github_token }}
```

Comment:
Do we need to add `fail_on_error: true`?

Reply:
Also we might want to use `filter_mode: 'nofilter'`

Reply:
Looking at the action closer, it looks like it uses "reviewdog" (which I was unfamiliar with) to run brakeman. [This action](https://github.com/devmasx/brakeman-linter-action) looks better for our use case.

Reply:
Ok awesome, adding that in now